oclint-rules/include/oclint/AbstractASTRuleBase.h

```diff
@@ -3,6 +3,7 @@
 
 #include <clang/AST/AST.h>
 
+#include "clang/Lex/Lexer.h"
```

Comment:
Can you use system include instead? Also let's align this one with line number 4, and add a new line break between system headers and our headers.